deploy/bin/build

```diff
@@ -30,8 +30,21 @@ packer_config() {
 }
 
 cmd_app() {
+  local docker_worker_tgz=/tmp/docker-worker-$$.tgz
   # Package up the node app.
-  tar --exclude='./node_modules' --exclude='./*.pub' --exclude='./build/' --exclude='./deploy' --exclude='.test' --exclude='.git' --exclude='.vagrant' --exclude='./.DS_Store' --exclude='./vagrant/' --exclude='app.box' -zcvf docker-worker.tgz .
+  tar \
+    --exclude='./node_modules' \
```

Comment:
why not just `--exclude=./docker-worker.tgz`?

Reply:
That was my first try, but if the file doesn't exist beforehand, tar ignores the `--exclude` option.

Reply:
Oh right.. *recalls vague memories from work at Zmanda*